pkg/service/snapshot_service.go

```diff
@@ -72,13 +72,13 @@ func (svc *SnapshotService) GetSnapshotById(id string) (*nimbleos.Snapshot, erro
 	return snapshotResp, nil
 }
 
-// GetSnapshotByName - method returns a pointer "Snapshot"
-func (svc *SnapshotService) GetSnapshotByName(name string) (*nimbleos.Snapshot, error) {
+// GetSnapshotByName - method returns a list of pointers to "Snapshot"
+func (svc *SnapshotService) GetSnapshotByName(volName string) ([]*nimbleos.Snapshot, error) {
```

Comment:
The function name is confusing.  It should be GetSnapshotsByVolumeName.

Reply:
This cannot be handled with this review. please don't mix.

Reply:
There is going to be next PR generated from build . you should expect the fix there .

Reply:
I don't understand the "don't mix" comment.  If generated things are out of bounds, what is left to review?  Anyway, if you are saying the change is coming in the next PR, that is fine with me.

Reply:
The fix is only for specific issues. This PR doesn't contain any changes related to the snapshot.

Reply:
Maybe the snapshot changes were also included by mistake.